docs/book/consuming-podcasts.md

```diff
@@ -0,0 +1,106 @@
+# Consuming Podcast Feeds
+
+Several extensions for podcast namespaces are included and enabled by default
+when importing a feed through laminas-feed's Reader subcomponent. Podcast feeds
+inherit most of their properties from RSS; refer to the "Consuming
+RSS Feeds" documentation for details.
```

Comment:
The introduction of this page should contain an explanation and a link to "Podcast Index" otherwise it will be difficult to follow the rest of the content.